packages/edit-widgets/src/store/actions.js

```diff
@@ -92,6 +107,13 @@ export const saveWidgetAreas = ( widgetAreas ) => async ( {
 	}
 };
 
+/**
+ * Converts all the blocks from a widget area specified by ID into widgets,
+ * and submits a batch request to save everything at once.
+ *
+ * @param {number} widgetAreaId ID of the widget area to process.
```

Comment:
I think the type of the id should be a string?

Reply:
Here's me being lazy with commits
```suggestion
 * @param {string} widgetAreaId ID of the widget area to process.
```